--- a/out/prod/extension-prod.js
+++ b/out/prod/extension-prod.js
@@ -130174,7 +130160,6 @@
         }
     }
     abTester_1.abTester.init(isFirstExtensionRun);
-    await extensionUtil_1.ExtensionUtil.updateExecutablePermissionsAsync();
     await launcher_1.Launcher.setup(false, !(await downloader_1.installFileExistsAsync()));
     activationEvent.markTime(telemetryStrings_1.TelemetryPropertyNames.EXTENSION_ACTIVATION_LAUNCHER_SETUP_COMPLETE);
     setupRpcFilters(rpcClientFactory_1.rpcClientFactory());
